test/parallel/test-util-inspect.js

```diff
@@ -690,7 +690,7 @@ assert.strictEqual(util.inspect(-5e-324), '-5e-324');
 
 {
   const tmp = Error.stackTraceLimit;
-  Error.stackTraceLimit = 0;
+  Object.defineProperty(Error, 'stackTraceLimit', { value: 0, enumerable: false });
```

Comment:
Do we need this change?

Reply:
Otherwise, `stackTraceLimit` will appear in this test and make the expectation noisy:
```js
  assert.strictEqual(
    inspect({
      get foo() { throw Error; }
    }, { getters: true }),
    '{ foo: [Getter: <Inspection threw ([Function: Error])>] }' // stackTraceLimit: 0 will appear here otherwise
  );
  ```
I think the test is relevant (it covers an uncommon value), but `stackTraceLimit` leaking into the output shouldn’t affect the expectation.

Reply:
nit: Adding a comment explaining that would be nice

Reply:
comment added in dd6be927896a2de1804d82894a74238107ad075e